### DISCUSSION TITLE
PBPotentiometer on Teensy not full

### DISCUSSION THREAD

**Discussion by gringow831** [tttapa/Control-Surface#687] [General]
Hi,

I am working with a TeensyLC and have been testing what I have done with my previous Leonardo Board. The PBPotentiometer is not working in full, it does start working after the 25%/30% while on the Leonardo worked in full from 0% is there something I am missing?

#include <Encoder.h>
#include <Control_Surface.h>

USBMIDI_Interface usbmidi;

Bank<3> bank(8); // A bank with four channels, and 2 bank settings

IncrementDecrementSelector<3> selector = {
  bank, {2, 3},  Wrap::Wrap,
};

Bankable::PBPotentiometer faders[] = {

  {{bank, BankType::CHANGE_CABLENB}, 17, {MCU::VOLUME_7, CABLE_1,}},
  {{bank, BankType::CHANGE_CABLENB}, 16, {MCU::VOLUME_8, CABLE_2,}},

  };
 
void setup() {
  RelativeCCSender::setMode(MACKIE_CONTROL_RELATIVE);
  Control_Surface.begin();
}

void loop() { // Refresh all inputs
  Control_Surface.loop();

}

**Comment by Veroledez** [comment]
Hola, con teensy lee el potenciometro solo con analogread, ve cual es el minimo y máximo, luego usa la funcion maping

**Reply by gringow831** [reply]
La diferencia ha estado en que estaba acostumbrado al Arduino Leonardo. Con Teensy no se debe aplicar 5v sino 3.3V.
El potenciometro empieza a actuar cuando llegas a 3.3v hasta 0v dejando 1.7V sin lectura
